src/main/java/com/example/_3dgraphics/Camera.java

```diff
@@ -1,8 +1,9 @@
 package com.example._3dgraphics;
-
+import com.example._3dgraphics.math.Vec4d;
 import com.example._3dgraphics.math.Matrix4x4;
 import com.example._3dgraphics.math.Vec4d;
 
+
```

Comment:
Remove empty line

Reply:
remove empty line by yourself, bitch!!!!!!!

Reply:
Get the fuck off you dumb peice of shit!!!!!

Reply:
I fucked your mother twice!